vavr/src/test/java/io/vavr/collection/CharSeqTest.java

```diff
@@ -4017,4 +4017,9 @@ public void shouldConvertToShortUsingRadix() {
     public void shouldNotConvertToShortUsingRadixGivenEmptyCharSeq() {
         CharSeq.empty().toShort(2);
     }
+
+    @Test
+    public void shouldCastTheCharSeqIntoJavaLangComparable() {
+        assertThat(CharSeq.empty()).isInstanceOf(Comparable.class);
```

Comment:
Please move the test to the `// -- compateTo(Object)` section and add the following three tests. It is good to have them too:

1. `<`

```java
@Test
public void shouldCompareToWhenLessThan() {
    assertThat(CharSeq.of("a").compareTo(CharSeq.of("b"))).isEqualTo(-1);
}
```

2. `>`

```java
@Test
public void shouldCompareToWhenLessThan() {
    assertThat(CharSeq.of("b").compareTo(CharSeq.of("a"))).isEqualTo(1);
}
```

3. `==`

```java
@Test
public void shouldCompareToWhenLessThan() {
    assertThat(CharSeq.of("a").compareTo(CharSeq.of("a"))).isEqualTo(0);
}
```